elastic/datadog_checks/elastic/elastic.py

```diff
@@ -238,6 +240,20 @@ def _get_index_metrics(self, admin_forwarder, version, base_tags):
             tags = base_tags + ['index_name:' + idx['index']]
             for metric, desc in iteritems(index_stats_for_version(version)):
                 self._process_metric(index_data, metric, *desc, tags=tags)
+        self._get_index_search_stats(admin_forwarder, base_tags)
```

Comment:
👍 I think for now this is fine, although like you said once it gets more complex later, I think it would make sense to have that extra method/function call `_get_index_metrics()` and then `_get_index_search_stats()` (and any other index-related metrics functions).

Reply:
Yea, I think my threshold for that is 3. I.e. if we add a 3rd part to `_get_index_metrics`, then I would turn that method into a proper "integration" method:

```python
def _get_index_metrics(self, ...):
    self._collect_cat_index(...)
    self._get_index_search_stats(...)
    self._get_some_new_amazing_index_metrics(...)
```